app/src/styles.css

```diff
@@ -70,6 +65,22 @@ button {
   display: inline-block;
   font-size: 1rem;
   width: 100%;
```

Comment:
When deployed, this doesn't seem to do anything and just shows `width: 75px` in dev tools.

With that being said, one more note, you use `rem` except for the width. Converting `width` to `rem` might be a good idea! On my screen `5rem` seems to look the exact same as `75px`.